src/options.ts

```diff
@@ -149,7 +148,7 @@ export async function loadOptions(
   options.preset =
     presetOverride ||
     (layers.find((l) => l.config.preset)?.config.preset as string) ||
-    defaultPreset;
+    (detectTarget({ static: !options.build }) ?? "node-server");
```

Comment:
I am thinking since we didn't release nitropack yet, would it made sense that we introduced `static: boolean` flag instead of top level `build` flag? It is mainly for sake of static support we introduced flag...

Reply:
yes, that works 👍

Reply:
you still prefer top-level static? very happy to change it if, on reflection, you feel that's the best.